### ISSUE TITLE
Dockerfile webapp

### ISSUE THREAD

**Issue by alebupal** [mattrayner/docker-lamp#44]
Hello, I am creating a webapp with docker and I have several doubts. (https://github.com/alebupal/tvspy/tree/desarrollo)
For my app to work properly I need to create a database and run a .sh
My dockerfile file is the following:
https://github.com/alebupal/tvspy/blob/desarrollo/Dockerfile
and the .sh to execute is the following:
https://github.com/alebupal/tvspy/blob/desarrollo/include/comandos.sh

The problem is that once the run.sh my script is run to create the db and the cron is not executed.

In short, I want that when the container starts (PHP, MYSQL) a database is created and execute a .sh so that my application works.
Greetings and thanks for the docker!
